ui/components/ui/list-item/index.scss

```diff
@@ -12,6 +12,7 @@
   display: grid;
   grid-template-columns: 0fr repeat(11, 1fr);
   grid-template-areas:
+    'top  top      top      top      top      top      top      top      top   top   top   top'
```

Comment:
Following our conversation, I think you can get rid of grid and do it with Flex. We can follow the same pattern as what we have for multichainTokenList